### PR TITLE
Validate review returns the right content type

### LINKED ISSUE #257: The validation view does not return right headers
The initial method in django rest framework , which actually return the final response is processing by `finalize_response` which actually apply the default headers. Consequences the actual view does not return a response json/application.

Moreover the exceptions.Http404 seems to no exist anymore, and have to be replace by exceptions.NotFound

### PR DESCRIPTION
**Pull request by moumoutte**
## Review

This PR closes #257 

- [x] Rebase from master
- [x] Add Changelog entry.

### CODE REVIEW

**Comment by brunobord** [comment]
Looking ok, although, please:

* You must rebase your PR from master,
* And then, you'll be able to add an entry to the Changelog.

**Comment by brunobord** [comment]
this branch is on hold, and will be merged targetting the 1.0.1 release

**Comment by brunobord** [comment]
You're clear to rebase (again, sorry), now that the 1.0.0 has been released

**Comment by brunobord** [comment]
You still need to add an item to the changelog, please.

**Comment by moumoutte** [comment]
Done

File: formidable/views.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def get(self, request, **kwargs):
         try:
             formidable = self.get_formidable_object(kwargs)
         except Formidable.DoesNotExist:
-            raise exceptions.Http404()
+            raise exceptions.NotFound()
```
Comment (wo0dyn) [inline formidable/views.py]: Nitpick: I think we can only return the reference of the exception here `raise exceptions.NotFound`, not an instance.

Comment (moumoutte) [reply formidable/views.py]: I don't mind , just want to know why a thing is better than another :)

Comment (wo0dyn) [reply formidable/views.py]: Dunno. I've always told that if you're not passing any argument to the exception's constructor, then there is no point of raising an instance, but a reference to the exception class suffice. It was a nitpick anyway. We can leave it as is 😉

Comment (moumoutte) [reply formidable/views.py]: Finally, I think the code is actually transforming the class to an instance automatically.
In the final exception handler, we can see something like this:

https://github.com/encode/django-rest-framework/blob/master/rest_framework/views.py#L433

But, `isinstance` does not work on class itself.

```python
```

**Pull request merged**